cps/translations/it/LC_MESSAGES/messages.po

```diff
@@ -124,7 +124,7 @@ msgstr "La posizione del Certfile non è valida, per favore indica il percorso c
 #: cps/shelf.py:100 cps/shelf.py:161 cps/shelf.py:202 cps/shelf.py:260
 #: cps/shelf.py:309 cps/shelf.py:338 cps/shelf.py:368 cps/shelf.py:392
 msgid "Settings DB is not Writeable"
-msgstr ""
+msgstr "I parametri del DB non sonos scrivibili"
```

Comment:
there's a typo.  
The correct phrase is: "I parametri del DB non sono scrivibili"

Reply:
:-) Yes, sorry...

Reply:
Now patch-15 - typo line 127 solved.
Thank you.